pkgs/development/python-modules/getmac/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, buildPythonPackage, fetchFromGitHub, pytest }:
+
+buildPythonPackage rec {
+  pname = "getmac";
+  version = "0.8.2";
+
+  src = fetchFromGitHub {
+    owner = "GhostofGoes";
+    repo = "getmac";
+    rev = version;
+    sha256 = "08d4iv5bjl1s4i9qhzf3pzjgj1rgbwi0x26qypf3ycgdj0a6gvh2";
+  };
+
+  doCheck = true;
+  checkInputs = [ pytest ];
```

pulled from their tox.ini, but we aren't too concerned about coverage and a few other metrics:

```suggestion
  checkInputs = [ pytest ];
  checkPhase = ''
    pytest
  '';
```
```